src/main/java/frc/robot/RobotContainer.java

```diff
@@ -196,7 +204,7 @@ private void configureButtonBindings() {
         JoystickButton btnLED = new JoystickButton(altJoystick, 5);
 
         driveSubsystem.setDefaultCommand(new RunCommand(
-            () -> driveSubsystem.drive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7),
+            () -> driveSubsystem.joystickDrive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7),
```

Comment:
Question: Is the *.7 because the robot moves too fast at full speed?

Reply:
Yeah, it was a little too fast so we added it with the intent of looking at it again once the drivers started practicing.